zend_abstract_interface/config/config_decode.c

```diff
@@ -164,16 +164,16 @@ static bool zai_config_decode_map(zai_str value, zval *decoded_value, bool persi
                     data++;
                 }
 
-                // Handle case without a colon (standalone key)
-                if (!has_colon && key_end) {
+                // Handle case without a colon (standalone key) only when lowercase is true which is current use case
+                if (lowercase && !has_colon && key_end) {
```

Comment:
Let's add a second boolean parameter, `allow_key_only` or such, and set it to true too for this case. Otherwise this will probably be confusing.

Reply:
Aaaaahh I not me adding a boolean `set` but then changing my mind to this instead, got it clarity first always.